ext/gd/tests/avif_decode_encode.phpt

```diff
@@ -58,8 +58,8 @@ gd
 
     // Note we could also forgive a certain number of pixel differences.
     // With the current test image, we just didn't need to.
-    echo 'How many pixels are different in the two images? ';
-    print_r(calc_image_dissimilarity($img, $img_from_avif));
+    echo 'What is the mean squared error of the two images? ';
+    print_r(mse($img, $img_from_avif));
```

Comment:
Aside, why is this using `print_r` for an int? Might as well fold this into the prior echo call.